envoy/tox.ini

```diff
@@ -12,6 +12,12 @@ platform = linux|darwin|win32
 deps =
     ../datadog_checks_base
     -rrequirements-dev.txt
+setenv =
+    FLAVOR=default
```

Comment:
what's a `FLAVOR`?

Reply:
This is for when I figure out how to connect to Envoy via Istio. `default` is just Envoy itself.

Reply:
can you elaborate what's the big design here? I've spent a fair amount of time getting rid of "flavors" in the previous CI...

Reply:
Istio would involve a whole different set of docker and config files. Flavor here doesn't mean version, as it used to. I can remove this if you don't find it valuable to test Envoy from Istio.